core-android/src/main/java/org/hisp/dhis/client/sdk/android/api/persistence/PersistenceModuleImpl.java

```diff
@@ -317,6 +317,7 @@ public boolean deleteAllTables() {
                 enrollmentStore.deleteAll() &&
                 trackedEntityInstanceStore.deleteAll() &&
                 trackedEntityAttributeValueStore.deleteAll() &&
-                optionSetStore.deleteAll();
```

Comment:
@xurxodev this method is called in a logout, and in a new conexion.
Is necessary modify this method? maybe we can use this method in the future to remove the entire sdk database, and i think the problem is in other place( maybe are we login more than one time?)
We can talk about it tomorrow,

https://github.com/EyeSeeTea/dhis2-android-sdk/blob/2.25-EyeSeeTea_temp_changes/core-android/src/main/java/org/hisp/dhis/client/sdk/android/api/D2.java#L347

https://github.com/EyeSeeTea/dhis2-android-sdk/blob/2.25-EyeSeeTea_temp_changes/core-android/src/main/java/org/hisp/dhis/client/sdk/android/user/CurrentUserInteractorImpl.java#L116